docs/staticAnalysis.md

```diff
@@ -102,6 +118,12 @@ To set it up, [install node.js](https://nodejs.org/en/download/) if necessary (v
 # Remember to use the correct tool version
 npm install -g eslint@{version}
 ```
+The plugin for IntelliJ can be found  [here](https://plugins.jetbrains.com/idea/plugin/7494-eslint).
+
+#### Configuring ESLint for IntelliJ
+
+1. Refer to [this guide](https://github.com/idok/eslint-plugin/blob/master/README.md#settings) to set up the ESLint plugin.
+1. Set the `Rules directory` to the location of `teammates-eslint.yml`.
```

Comment:
`Builtin rules path is not a directory. Path C:\Users\leeyimin\Documents\teammates\node_modules\eslint\bin\eslint.js not found in project` I get this error when I try running the plugin. Is the file not a config file?

Maybe you should reference the file by `static-analysis/teammates-eslint.yml`, in case the reader isn't sure which folder you are referring to.